app/models/webhooks/slack/base.rb

```diff
@@ -57,6 +58,15 @@ def eventable
     @eventable ||= event.eventable
   end
 
+  def proposal_name(event)
+    # Access proposal name as new events are added
+    if event.is_a?(Motion)
```

Comment:
I think we want a proper case statement here
```
case event
when Motion then event.name
when Vote then event.motion_name
end
```

Also motions / votes are again models, not events